src/scripts/navigation-line.js

```diff
@@ -449,8 +449,8 @@ const NavigationLine = (function ($) {
       .addClass("h5p-progressbar-part-selected")
       .attr('aria-selected', true)
       .siblings()
-        .removeClass("h5p-progressbar-part-selected")
-        .attr('aria-selected', false);
+      .removeClass("h5p-progressbar-part-selected")
+      .attr('aria-selected', false);
```

Comment:
Is there a reason you removed the indents here?

Reply:
It's for the code formatting norme, you might be right since the indented code concerns only the siblings

Reply:
From [this example](https://contribute.jquery.org/style-guide/js/#chained-method-calls) it looks like there should be an extra level of indentation when the context changes.